diff --git a/node_modules/openid/openid.js b/node_modules/openid/openid.js
index 3a20a6e..228162b 100644
--- a/node_modules/openid/openid.js
+++ b/node_modules/openid/openid.js
@@ -57,9 +57,17 @@ openid.RelyingParty = function(returnUrl, realm, stateless, strict, extensions)
   this.extensions = extensions;
 }
 
-openid.RelyingParty.prototype.authenticate = function(identifier, immediate, callback)
+openid.RelyingParty.prototype.authenticate = function(identifier, immediate, callback, queryString)
 {
-  openid.authenticate(identifier, this.returnUrl, this.realm, 
+  var callbackUrl
+
+  if (queryString === undefined) {
+    callbackUrl = this.returnUrl;
+  } else {
+    callbackUrl = this.returnUrl + "?" + queryString;
+  }
+
+  openid.authenticate(identifier, callbackUrl, this.realm,
       immediate, this.stateless, callback, this.extensions, this.strict);
 }
 
